.github/workflows/release-publish.yml

```diff
@@ -47,20 +39,72 @@ jobs:
       - name: docker login
         uses: docker/login-action@v2.2.0
         with:
-          username: flowforge
+          username: flowfuse
           password: ${{ secrets.DOCKER_HUB_PASSWORD }}
-      - name: Build and push FlowForge Device Agent container
+      - name: Build and push FlowFuse Device Agent container
         uses: docker/build-push-action@v4.0.0
         with:
           context: docker
           file: docker/Dockerfile
           tags: ${{ steps.meta.outputs.tags }}
           push: true
           platforms: linux/amd64, linux/arm64, linux/arm/v7
+      - name: Publish README.md to Docker hub
+        uses: peter-evans/dockerhub-description@v3
+        with:
+          repository: flowfuse/device-agent
+          username: flowfuse
+          password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+          readme-filepath: ./docker/README.md
+  publish_legacy:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
```suggestion
  publish_legacy:
    needs: [publish]
    runs-on: ubuntu-latest
    steps:
```

Reply:
@hardillb this should ensure the legacy build only runs once the publish of the new package has happened; the legacy container uses the new package internally.

Reply:
Ahh, yes because the legacy container is still using the new package. makes sense.